runners-manager/main.py

```diff
@@ -17,34 +18,32 @@ def maintain_number_of_runner(runner_m: RunnerManager):
             f"offline: {len([e for e in runners['runners'] if e['status'] == 'offline'])}"
         )
 
-        for elem in runners['runners']:
-            runner_m.update_runner(elem)
+        print(runners)
```

Comment:
We need to start to think about adding a real logger instead of using print statements.